src/frameworks/foundation/ns_string.rs

```diff
@@ -435,6 +435,18 @@ pub const CLASSES: ClassExports = objc_classes! {
     new_string
 }
 
+- (id)lastPathComponent { // NSString*
+    // TODO: avoid copying
```

Comment:
I don't think we can actually avoid copying in this case, so you can remove this comment.

Reply:
Sorry, what am I saying, of course we can avoid copying for the `to_rust_string` part (eventually). Ignore me.